From 23deab5b588c1b15a8ede46080bf7e5950b86f88 Mon Sep 17 00:00:00 2001
From: Steev Klimaszewski <steev@kali.org>
Date: Mon, 20 Feb 2023 00:55:11 -0600
Subject: [PATCH 230/230] arm64: configs: laptop: Add latest bits

This makes a couple of things modular, and adds a needed option for the
audio to work.
---
 arch/arm64/configs/laptop_defconfig | 24 +++++++++++++-----------
 1 file changed, 13 insertions(+), 11 deletions(-)

diff --git a/arch/arm64/configs/laptop_defconfig b/arch/arm64/configs/laptop_defconfig
index 02fe3b9424e..903ac5a6029 100644
--- a/arch/arm64/configs/laptop_defconfig
+++ b/arch/arm64/configs/laptop_defconfig
@@ -558,12 +558,12 @@ CONFIG_BAYCOM_PAR=m
 CONFIG_YAM=m
 CONFIG_CAN=m
 CONFIG_BT=m
-CONFIG_BT_RFCOMM=y
+CONFIG_BT_RFCOMM=m
 CONFIG_BT_RFCOMM_TTY=y
-CONFIG_BT_BNEP=y
+CONFIG_BT_BNEP=m
 CONFIG_BT_BNEP_MC_FILTER=y
 CONFIG_BT_BNEP_PROTO_FILTER=y
-CONFIG_BT_HIDP=y
+CONFIG_BT_HIDP=m
 CONFIG_BT_6LOWPAN=m
 CONFIG_BT_LEDS=y
 CONFIG_BT_HCIBTUSB=m
@@ -589,7 +589,7 @@ CONFIG_RXKAD=y
 CONFIG_CFG80211=m
 CONFIG_MAC80211=m
 CONFIG_MAC80211_MESH=y
-CONFIG_RFKILL=y
+CONFIG_RFKILL=m
 CONFIG_RFKILL_INPUT=y
 CONFIG_NET_9P=m
 CONFIG_NET_9P_VIRTIO=m
@@ -643,6 +643,7 @@ CONFIG_ARM_SCMI_TRANSPORT_SMC_ATOMIC_ENABLE=y
 CONFIG_ARM_SCPI_PROTOCOL=y
 CONFIG_DMI_SYSFS=y
 CONFIG_FW_CFG_SYSFS=m
+CONFIG_QCOM_TEE_UEFISECAPP=y
 CONFIG_GOOGLE_FIRMWARE=y
 CONFIG_GOOGLE_COREBOOT_TABLE=m
 CONFIG_GOOGLE_MEMCONSOLE_COREBOOT=m
@@ -1410,13 +1411,13 @@ CONFIG_PINCTRL_SINGLE=y
 CONFIG_PINCTRL_MSM=y
 CONFIG_PINCTRL_QCOM_SPMI_PMIC=y
 CONFIG_PINCTRL_QCOM_SSBI_PMIC=y
-CONFIG_PINCTRL_SC7280_LPASS_LPI=y
+CONFIG_PINCTRL_SC7280_LPASS_LPI=m
 CONFIG_PINCTRL_SC8280XP=y
 CONFIG_PINCTRL_SM8250=m
-CONFIG_PINCTRL_SM8250_LPASS_LPI=y
-CONFIG_PINCTRL_SM8450_LPASS_LPI=y
-CONFIG_PINCTRL_SC8280XP_LPASS_LPI=y
-CONFIG_PINCTRL_LPASS_LPI=y
+CONFIG_PINCTRL_SM8250_LPASS_LPI=m
+CONFIG_PINCTRL_SM8450_LPASS_LPI=m
+CONFIG_PINCTRL_SC8280XP_LPASS_LPI=m
+CONFIG_PINCTRL_LPASS_LPI=m
 CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_EXAR=m
 CONFIG_GPIO_MB86S7X=m
@@ -2383,6 +2384,7 @@ CONFIG_QCOM_CLK_RPMH=y
 CONFIG_MSM_GCC_8916=y
 CONFIG_MSM_MMCC_8996=y
 CONFIG_SC_DISPCC_8280XP=y
+CONFIG_SC_LPASSCSR_8280XP=y
 CONFIG_SC_GCC_8180X=y
 CONFIG_SC_GPUCC_8280XP=y
 CONFIG_SDM_CAMCC_845=m
@@ -2596,6 +2598,7 @@ CONFIG_NFS_SWAP=y
 CONFIG_NFS_V4_1=y
 CONFIG_NFS_V4_2=y
 CONFIG_NFS_FSCACHE=y
+CONFIG_NFS_V4_2_READ_PLUS=y
 CONFIG_NFSD=m
 CONFIG_NFSD_V3_ACL=y
 CONFIG_NFSD_V4=y
@@ -2689,7 +2692,6 @@ CONFIG_CRYPTO_USER=m
 # CONFIG_CRYPTO_MANAGER_DISABLE_TESTS is not set
 CONFIG_CRYPTO_PCRYPT=m
 CONFIG_CRYPTO_TEST=m
-CONFIG_CRYPTO_ECDH=y
 CONFIG_CRYPTO_AES=y
 CONFIG_CRYPTO_ANUBIS=m
 CONFIG_CRYPTO_BLOWFISH=m
@@ -2715,6 +2717,7 @@ CONFIG_CRYPTO_CRC32C=y
 CONFIG_CRYPTO_LZ4=m
 CONFIG_CRYPTO_LZ4HC=m
 CONFIG_CRYPTO_ANSI_CPRNG=m
+CONFIG_CRYPTO_DRBG_MENU=y
 CONFIG_CRYPTO_USER_API_HASH=m
 CONFIG_CRYPTO_USER_API_SKCIPHER=m
 CONFIG_CRYPTO_USER_API_RNG=m
@@ -2775,7 +2778,6 @@ CONFIG_TRACER_SNAPSHOT=y
 CONFIG_BLK_DEV_IO_TRACE=y
 CONFIG_IO_STRICT_DEVMEM=y
 CONFIG_NOTIFIER_ERROR_INJECTION=m
-CONFIG_FUNCTION_ERROR_INJECTION=y
 CONFIG_TEST_USER_COPY=m
 CONFIG_TEST_BPF=m
 CONFIG_TEST_FIRMWARE=m
-- 
2.39.0

